src/Credentials/CredentialsUtils.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Aws\Credentials;
+
+final class CredentialsUtils
```

Comment:
This is not in the specification for imds endpoint configuration, but checking whether or not we should do this.

Reply:
Checked and I think this is okay.  We can always loosen restrictions later, but probably a good move from a security standpoint.